File: actividad hrencias/herncia/Vehiculo.py
class Vehiculo:
    def __init__(self, fecha_fabricacion, vin_chasis, vin_motor):
        self.__fecha_fabricacion = fecha_fabricacion
        self.__vin_chasis = vin_chasis
        self.__vin_motor = vin_motor
    
    def obtener_fecha_fabricacion(self):
        return self.__fecha_fabricacion
    
    def obtener_vin_chasis(self):
        return self.__vin_chasis
    
    def obtener_vin_motor(self):
        return self.__vin_motor

File: actividad hrencias/herencia/src/Vehiculo.java
public class Vehiculo {
    // Atributos privados (encapsulación)
    private String fechaFabricacion;
    private String vinChasis;
    private String vinMotor;
    
   
    public Vehiculo(String fechaFabricacion, String vinChasis, String vinMotor) {
        this.fechaFabricacion = fechaFabricacion;
        this.vinChasis = vinChasis;
        this.vinMotor = vinMotor;
    }
    
    
    public String obtenerFechaFabricacion() {
        return this.fechaFabricacion;
    }
    
    
    public String obtenerVinChasis() {
        return this.vinChasis;
    }
    
    
    public String obtenerVinMotor() {
        return this.vinMotor;
    }
}